addons/l10n_in_pos/models/pos_order.py

```diff
@@ -24,3 +24,15 @@ def _prepare_account_move_line(self, line, partner_id, current_company, currency
                     'product_uom_id': line.product_id.uom_id.id
                     })
         return res
+
+    def _prepare_invoice_vals(self):
+        vals = super()._prepare_invoice_vals()
+        if self.session_id.config_id.invoice_journal_id.company_id.country_id.code == 'IN':
```

Comment:
you can just do: if self.session_id.company_id.country_id.code == 'IN':

Reply:
okay